src/lib_utf8.c

```diff
@@ -0,0 +1,25 @@
+/*
+** utf8 library.
+** Copyright (C) 2019 moonjit developers. https://github.com/moonjit/moonjit
+*/
+
+#define LUA_LIB
+
+#include "lua.h"
+#include "lauxlib.h"
+#include "lualib.h"
+
+#include "lj_arch.h"
+#include "utf8.h"
+
+/* ------------------------------------------------------------------------ */
+
+LUALIB_API int luaopen_utf8(lua_State *L)
```

Comment:
This needs a mention in the documentation, i.e. not just the function but also the fact that we have the UTF8 library now.